sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/V2ExpressionUtils.scala

```diff
@@ -182,7 +182,7 @@ object V2ExpressionUtils extends SQLConfHelper with Logging {
             ApplyFunctionExpression(scalarFunc, arguments)
           case _ =>
             throw new AnalysisException(
-              errorClass = "_LEGACY_ERROR_TEMP_3055",
+              errorClass = "SCALAR_FUNCTION_NOT_FULLY_IMPLEMENTED",
               messageParameters = Map("scalarFunc" -> scalarFunc.name()))
```

Comment:
Since `scalarFunc.name` is an id, let's quote it by `toSQLId()`

Reply:
Replaced with 

```
messageParameters = Map("scalarFunc" -> toSQLId(scalarFunc.name())))
```